test/components/plots/clusteredBarPlotTests.ts

```diff
@@ -80,7 +80,7 @@ describe("Plots", () => {
 
       // check that clustering is correct
       var innerScale = (<any>renderer)._makeInnerScale();
-      var off = innerScale.scale("_0");
+      var off = innerScale.scale("0");
       var width = xScale.rangeBand() / 2;
       assert.closeTo(TestMethods.numAttr(bar0, "x") + TestMethods.numAttr(bar0, "width") / 2, xScale.scale(bar0X) - width + off, 0.01
```

Comment:
I agree we can come back for it later. I just wanted to note that the test would probably be less fragile if the bars positions were checked "fuzzily" relative to each other, rather than a strict check of the position value.